### DISCUSSION TITLE
Start an App

### DISCUSSION THREAD

**Discussion by andreasbaum123** [BimmerGestalt/AAIdrive#587] [Ideas]
Hello,

I would like to start an App if AAIdrive connect.

**Comment by hufman** [comment]
What would you like to start? The easiest thing would be for Tasker to run something when your car's Bluetooth connects. The Android Intent system can be used too, MyBMW broadcasts a connection announcement to any app on the phone when it connects. You could also use a voice assistant to run the app without opening the phone.
I have no idea how starting an app works if the screen is off, though.
